src/main/java/com/stripe/param/PersonUpdateParams.java

```diff
@@ -1804,6 +1819,198 @@ public Builder addAllFile(List<String> elements) {
     }
   }
 
+  @Getter
+  public static class RegisteredAddress {
+    /** City, district, suburb, town, or village. */
+    @SerializedName("city")
+    Object city;
```

Comment:
Should that all be objects? Should we share Address here?

Reply:
Good question!

> Should that all be objects? 

We use Object here since it's emptyable. Below we have overloaded methods to set one or the other:

```java

        /** City, district, suburb, town, or village. */
        public Builder setCity(String city) {
          this.city = city;
          return this;
        }

        /** City, district, suburb, town, or village. */
        public Builder setCity(EmptyParam city) {
          this.city = city;
          return this;
        }

```

This is consistent with other uses of Address eg. https://github.com/stripe/stripe-java/blob/41530ea4/src/main/java/com/stripe/param/AccountUpdateParams.java#L721

> Should we share Address here?

We don't share Address in params, just in the response type.